### PR TITLE
add all americas gcp locations

### PR DESCRIPTION
**Pull request by nater1067**
This PR adds the remaining Americas GCP locations and reorders them in the order listed in the GCP docs here:
[https://cloud.google.com/about/locations](https://cloud.google.com/about/locations)

### CODE REVIEW

File: src/config_ctrl.ts [hunk 1]
```diff
@@ -50,13 +50,17 @@ export class BigQueryConfigCtrl {
...
+      { text: 'Iowa (us-central1)', value: 'us-central1' },
+      { text: 'South Carolina (us-east1)', value: 'us-east1' },
+      { text: 'Northern Virginia (us-east4)', value: 'us-east4' },
       {
         text: 'Montréal (northamerica-northeast1)',
         value: 'northamerica-northeast1',
       },
-      { text: 'South Carolina (us-east1)', value: 'us-east1' },
-      { text: 'Northern Virginia (us-east4)', value: 'us-east4' },
       { text: 'São Paulo (southamerica-east1)', value: 'southamerica-east1' },
```
Comment (nater1067) [inline src/config_ctrl.ts:64]: Reordered these per documentation: https://cloud.google.com/about/locations

File: src/partials/query.editor.html [hunk 2]
```diff
@@ -168,10 +168,14 @@
...
+                  <option value=us-west1>Oregon (us-west1)</option>
                   <option value=us-west2>Los Angeles (us-west2)</option>
-                  <option value=northamerica-northeast1>Montréal (northamerica-northeast1)</option>
+                  <option value=us-west3>Salt Lake City (us-west3)</option>
+                  <option value=us-west4>Las Vegas (us-west4)</option>
+                  <option value=us-central1>Iowa (us-central1)</option>
                   <option value=us-east1>South Carolina (us-east1)</option>
                   <option value=us-east4>Northern Virginia (us-east4)</option>
+                  <option value=northamerica-northeast1>Montréal (northamerica-northeast1)</option>
                   <option value=southamerica-east1>São Paulo (southamerica-east1)</option>
```
Comment (nater1067) [inline src/partials/query.editor.html:179]: Reordered Montreal per documentation: https://cloud.google.com/about/locations

File: src/query_ctrl.ts [hunk 3]
```diff
@@ -64,14 +64,15 @@ export class BigQueryQueryCtrl extends QueryCtrl {
...
+      { text: 'Los Angeles (us-west2)', value: 'us-west2' },
       { text: 'Salt Lake City (us-west3)', value: 'us-west3' },
-      { text: 'São Paulo (southamerica-east1)', value: 'southamerica-east1' },
+      { text: 'Las Vegas (us-west4)', value: 'us-west4' },
+      { text: 'Iowa (us-central1)', value: 'us-central1' },
       { text: 'South Carolina (us-east1)', value: 'us-east1' },
+      { text: 'Northern Virginia (us-east4)', value: 'us-east4' },
+      { text: 'Montréal (northamerica-northeast1)', value: 'northamerica-northeast1' },
+      { text: 'São Paulo (southamerica-east1)', value: 'southamerica-east1' },
       // Europe
```
Comment (nater1067) [inline src/query_ctrl.ts:76]: Reordered these per documentation: https://cloud.google.com/about/locations

**Comment by ofir5300** [comment]
Great contribution @nater1067 
Merged locally to https://github.com/doitintl/bigquery-grafana/pull/358

**Pull request closed without merging**